assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -289,7 +289,10 @@ export default class LiveEditor {
       this.language &&
       LanguageDescription.matchLanguageName(languages, this.language, false);
 
-    const customKeymap = [{ key: "Escape", run: exitMulticursor }];
+    const customKeymap = [
+      { key: "Escape", run: exitMulticursor },
+      { key: "Ctrl-Enter", run: insertBlankLine },
```

Comment:
We actually use ctrl+enter to "evaluate cell" on both Linux and Windows, so unfortunately this combination is taken. I'm not sure if there's a free combination involving Enter that would also be intuitive, and introducing a specific shortcut for this kind of action seems like too much :(

Reply:
Hmm, the clash is indeed unfortunate, but I respectfully disagree that having a shortcut for this is too much. 

The ability to start a new line without first moving to the end of the current line is table-stakes for any editor which auto-closes brackets/quotes etc. It's actually already built into CodeMirror (as well as IntelliJ, VSCode & Zed!) as Ctrl/Cmd+Enter, but that obviously also clashes with evaluate cell. 

Do we really need three (or more?) different shortcuts for "evaluate cell"?

Reply:
Evaluation is the most important feature of Livebook, and it has both productivity and security concerns, the three existing shortcuts look appropriate to me.

My only suggestion would be to use “alt+enter” (and its equivalent on Mac) but I don’t know if that’s possible.

Reply:
Alt+Enter would be fine, but for some unfathomable reason, CodeMirror binds it to "Find & Replace" by default. Would you be happy to remap it to something else? (VSCode, IntelliJ and Zed use Cmd+F on a Mac)

Reply:
That’s fine by me. :)

Reply:
Alternatively, I'd be happy with a three-key binding, say Cmd+Shift+Enter, but that also appears to be bound to "evaluate cell".

Reply:
`Alt/Opt + Enter` is fine. Cmd + F already opens "Find & Replace" and is the most common key binding, so I think we are good.

> Cmd+Shift+Enter, but that also appears to be bound to "evaluate cell".

FTR all of these combinations do something different. Cmd+Enter evaluates a single cell, Cmd+Shift+Enter evaluates all cells, and Shift+Enter evaluates + moves to the next cell (which is a common shortcut among notebook applications, that some users expect to work).

Reply:
Ah, ok, that's actually really cool -- thanks for the explanation! :) 
I've adjusted the PR to use Alt+Enter. We still have the issue with the tooltip not hiding by itself, and the new shortcut doesn't seem to work in Safari.

Reply:
I pushed a fix for the tooltips. Also, it works in Safari for me :)

Reply:
That's awesome! Thanks so much! :)